src/admin-views/base_admin_panel.php

```diff
@@ -0,0 +1,67 @@
+<div id="tribe_panel_base" class="ticket_panel panel_base" aria-hidden="false">
+	<div class="tribe_sectionheader ticket_list_container">
+		<div class="ticket_table_intro">
+			<span class="ticket_form_total_capacity">
+				Total Event Capacity:
+				<span id="ticket_form_total_capacity_value">
+					<?php
+					switch ( $total_tickets ) {
+						case -1:
+							?><i><?php esc_html_e( 'unlimited', 'event-tickets' ); ?></i><?php
```

Comment:
I don't personally feel super strongly about this, but some might say we should avoid `<i>` and use CSS to italicize (assuming we're not using `<em>` because we're not seeking to emphasize it as such)